### PR TITLE
Verify PV is available before add capacity

### PR DESCRIPTION
**Pull request by prsurve**
fixes: #3388

Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by yitzhak12 — DISMISSED** [review]
LGTM

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -103,6 +103,7 @@
 CEPHBLOCKPOOL_SC = "ocs-storagecluster-ceph-rbd"
 CEPHFILESYSTEM_SC = "ocs-storagecluster-cephfs"
 NOOBAA_SC = "openshift-storage.noobaa.io"
+LOCALSTORAGE_SC = "localblock"
```
Comment (sidhant-agrawal) [inline ocs_ci/ocs/constants.py:106]: There is an existing constant at [#L1026](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L1026) which we can reuse

**Review by yitzhak12 — DISMISSED** [review]
LGTM

**Review by ebenahar — APPROVED** [review]

**Comment by yitzhak12** [comment]
I ran 2 PR validation locally cause I had some issues with running the PR on Jenkins. I uploaded the logs to http://rhsqe-repo.lab.eng.blr.redhat.com/OCS/ocs-qe-bugs/pr-4364-test-logs/. Link to the cluster I used: https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/4354/

**Review by yitzhak12 — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]
LGTM

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**